staff/johana-ludwig/cv/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Johana Ludwig</title>
+</head>
+<body>
+    <header>
+        <h1>Johana Ludwig</h1>
+        <h2>Online Marketing y Frontend Development</h2>
+        <figure>
+            <img src="" alt="Foto de perfil Johana Ludwig">
+        </figure>
+    </header>
+
+    <main>
+    <section>
+        <table>
+            <tr>
+                <td></td>
+                <td>
+                    <header><h3>Hola, soy Johana Ludwig</h3></header>
+                        <ul>
+                            <li>Campañas de marketing online</li>
+                            <li>SEO y SEA</li>
+                            <li>Apasionada al desarrollo web</li>
+                        </ul>
+                </td>
+            </tr>               
+        </table>
+    </section>
+    <hr>
+    <section>
+        <h2>Experiencia laboral</h2>
+        <table>
+            <tr>
+                <td>10/2015 - actual</td>
+            </tr>
+            <tr>
+                <td></td>
+                <td>Chrono24 GmbH - Kollwitzstr. 47, 12654 Berlin</td>
+            </tr>                
+            <tr>
+                <td></td>
+                <td>
+                    <ul>
+                        <li>Un tarea en la empresa</li>
+                        <li>otra tarea en la empresa</li>
+                        <li>segunda tarea en la empresa</li>
+                    </ul>
+                </td>
+            </tr> 
+        </table>
+    </section>
+    
+    <section>
+        <h2>Estudios y formación</h2>
+            <table>
+                <tr>
+                    <td>Business Administration UHGG</td>
+                </tr>
+                <tr>
+                    <td>Skylab Coders</td>
+                </tr>
+            </table>
+    </section>
+</main>
+<hr>
+<footer>
+    <aside>
```

Comment:
Maybe the <aside> tag here is not necessary, because it's the only element inside the footer.